docs/core/diagnostics/eventpipe.md

```diff
@@ -19,7 +19,7 @@ EventPipe aggregates events emitted by runtime components - for example, the Jus
 
 The events are then serialized in the `.nettrace` file format and can be written directly to a file or streamed through a [diagnostic port](./diagnostic-port.md) for out-of-process consumption.
 
-To learn more about the EventPipe serialization format, refer to the [EventPipe format documentation](https://github.com/microsoft/perfview/blob/main/src/TraceEvent/EventPipe/EventPipeFormat.md).
+To learn more about the EventTrace format, refer to the [EventTrace format documentation](https://github.com/microsoft/perfview/blob/main/src/TraceEvent/EventPipe/NetTraceFormat.md).
```

Comment:
```suggestion
To learn more about the NetTrace format, see the [NetTrace format documentation](https://github.com/microsoft/perfview/blob/main/src/TraceEvent/EventPipe/NetTraceFormat.md).
```

Reply:
oh you are absolutely right, I looked too quickly :) `NetTrace` is the proper term.